url_gatekeeper/__init__.py

```diff
@@ -1,11 +1,19 @@
-from django.core.cache import get_cache
+try:
```

Comment:
Please add comment explaining which versions of Django fall into which case of the try-catch